src/app/components/InstructionModal.tsx

```diff
@@ -1,54 +0,0 @@
-import React from 'react';

```

Comment:
sonar reported a bug, decide to remove based on your yday feedback @athrael-soju 


<img width="1020" alt="image" src="https://github.com/athrael-soju/Iridium-AI/assets/1477453/ab4ee759-509a-4754-aeb4-a6534c70ba8f">